docs/get-started/onnx/index.md

```diff
@@ -0,0 +1,238 @@
+    epochs=20,
+    batch_size=128,
+    to_embedding_model=True,
+    input_size=(3, 224, 224),
+    layer_name='adaptiveavgpool2d_67',
+    freeze=False,
+)
+
+```
+
+We can now export the model to the ONNX format, using the `to_onnx` method provided
+by finetuner. The `onnx` package is required to use this functionality. Also, if
+you are using TensorFlow or PaddlePaddle you will need to install additional packages
+for the ONNX conversion. Specifically you will need 
+[tf2onnx](https://github.com/onnx/tensorflow-onnx) for TensorFlow and
+[paddle2onnx](https://github.com/PaddlePaddle/Paddle2ONNX) for PaddlePaddle.
+
+```bash
+pip install onnx
+pip install tf2onnx
+pip install paddle2onnx
+```
+
+Back to the ONNX conversion:
+
+```python
+from finetuner.tuner.onnx import to_onnx
+
+to_onnx(tuned_model, 'tuned-model.onnx', input_shape=[3, 224, 224], opset_version=13)
+```
+
+The ONNX exported model should be stored now at `tuned-model.onnx`.
+
+A helper function for validating the ONNX export is also provided:
+```python
+from finetuner.tuner.onnx import validate_onnx_export
+
+validate_onnx_export(tuned_model, 'tuned-model.onnx', input_shape=[3, 224, 224])
+```
+
+This function compares the outputs of the native model and its ONNX counterpart against
+the same random input. If the outputs differ an assertion error is raised.
+
+Now that we have our model exported and have verified that it has the same behaviour as the
+original, it's time to deploy it 🚀.
+
+
+## Deploying using the ONNX Encoder
+
+You have already finetune your own model and transfer it to onnx format. Let's start deploying in Jina flow. If you are not familiar with Jina-Hub or Jina flow, check this:
+[Use Hub Executor](https://docs.jina.ai/advanced/hub/use-hub-executor/)
+[Use Jina Flow](https://docs.jina.ai/fundamentals/flow/)
+
+Here are steps you can follow:
+
+### Add ONNXEncoder to Jina flow
+
+We already have onnx encoder on Jina-Hub, let's add ONNXEncoder to jina flow:
+
+using docker image:
+
+```python
+from jina import Flow
+
+f = Flow().add(uses='jinahub+docker://ONNXEncoder',
+                uses_with={'model_path': 'tuned-model.onnx'})
+```
+
+or using source code:
+
+```python
+from jina import Flow
+
+f = Flow().add(uses='jinahub://ONNXEncoder',
+                uses_with={'model_path': 'tuned-model.onnx'})
+```
+
+model_path is the path of onnx model you exported.
+
+### Complete the flow by adding indexer and starting the service
+
+```python
+from jina import DocumentArray, Executor, requests, Flow
+from typing import Optional
+import numpy as np
+
+
+class SimpleIndexer(Executor):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._dam = DocumentArray()
+
+    @requests(on='/index')
+    def index(self, docs: DocumentArray, **kwargs):
+        self._dam.extend(docs)
+
+    @requests(on='/search')
+    def search(self, docs: DocumentArray, **kwargs):
+        docs.match(self._dam)
+        
+
+f = Flow(port_expose=12345, 
```

Comment:
use single quotes `''`

Reply:
do you mean using single quotes to wrap these pieces of code?

Reply:
single quotes in strings, `'foo'` instead of `"foo"`